frontend/src/components/App/NotificationIcon.js

```diff
@@ -1,6 +1,11 @@
 import React, { PropTypes, Component } from 'react'
 
 class NotificationIcon extends Component {
+
+  static propTypes = {
+    unreadNotificationsCount: PropTypes.number
+  }
```

Comment:
if we did do this, we could change the top line to 

`export default class NotificationIcon extends Component`

which actually is kind of nice because the export default is clear at the top, rather than sometimes forgetting to add it at the bottom when making a new file